### ISSUE TITLE
Word 2010: Unable to use Table Navigation Commands within a Protected Document after restarting NVDA

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#6665]
STR:

1. Create a new Word document and add at least one table. Test to ensure that the navigation commands for moving to next column/row work as expected

2. Reopen the same document in Protected View; the best way to achieve this is most likely by sending this file to yourself (i.e. via email) to ensure that Windows marks the file as being downloaded from an external location

3. Restart NVDA and navigate back to the document; attempt to use the same commands as before

Expected: NVDA should read table content as it did previously

Actual: NVDA outputs that focus is not currently on a table, and the document must be re-opened before the commands can be used

**Comment by Adriani90** [comment]
@elliott94 are you still having this issue? I cannot reproduce it.

**Comment by elliott94** [comment]
Unfortunatley I don't have a copy of Office 2010 that I can test with right now - is anybody else able to reproduce?

**Comment by Adriani90** [comment]
Testing with NVDA 2019.1 and NVDA 2018.4.1 in MS Word 2010, I cannot reproduce this issue. If someone can reproduce it, Please comment on it and we will reopen it. I am closing this as works for me.
